elementary/monitor/dbt_project/macros/tests_recommendation/get_recommended_tests.sql

```diff
@@ -0,0 +1,40 @@
+{% macro get_recommended_tests(
+    depends_on_count=none,
+    dependant_on_count=none,
+    exposure_count=none,
+    critical_name_like_patterns=none,
+    table_types=none
+) %}
+    {# TODO: Decide if those should be OR or AND operators. #}
+    {% set query %}
+        select resource_name, source_name, test_namespace, test_name, timestamp_column
+        from {{ ref("pending_test_recommendations") }}
+        where 1=1
+
+        {% if depends_on_count %}
+            and depends_on_count >= {{ depends_on_count }}
+        {% endif %}
+
+        {% if dependant_on_count %}
+            and dependant_on_count >= {{ dependant_on_count }}
+        {% endif %}
+
+        {% if exposure_count %}
+            and exposure_count >= {{ exposure_count }}
+        {% endif %}
+
+        {% if critical_name_like_patterns %}
+            {% for name_like_pattern in critical_name_like_patterns %}
+                and resource_name like '{{ name_like_pattern }}'
```

Comment:
Should we use "ilike"?

Reply:
Everything is lower-cased in the models anyway so that'd just be less efficient for no reason.